chalice/app.py

```diff
@@ -692,8 +704,14 @@ def _validate_response(self, response):
     def _cors_enabled_for_route(self, route_entry):
         return route_entry.cors is not None
 
-    def _add_cors_headers(self, response, cors):
+    def _get_cors_headers(self, cors):
```

Comment:
Is the intent here just to copy the dictionary? 

You can just use `dict(cors.get_access_control_headers())` or `cors.get_access_control_headers().copy()`. No need to manually iterate.